.travis.yml

```diff
@@ -47,6 +47,22 @@ script:
   - cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr -DENABLE_TESTING=ON -Dsqlcipher=1 ..
   - make
   - ctest -V
+  - # AppImage generation
+  - cd ../build
+  - sudo apt-get -y install checkinstall
+  - sudo checkinstall --pkgname=app --pkgversion="1" --pkgrelease="1" --backup=no --fstrans=no --default --deldoc 
+  - mkdir appdir ; cd appdir
+  - dpkg -x ../app_1-1_amd64.deb . ; find .
+  - cp ./usr/share/applications/sqlitebrowser.desktop .
+  - cp ./usr/share/icons/hicolor/256x256/apps/sqlitebrowser.png .
+  - cd .. 
+  - wget -c "https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-x86_64.AppImage" 
+  - chmod a+x linuxdeployqt*.AppImage
+  - unset QTDIR; unset QT_PLUGIN_PATH ; unset LD_LIBRARY_PATH
+  - ldd ./appdir/usr/bin/sqlitebrowser
+  - ./linuxdeployqt*.AppImage ./appdir/usr/bin/sqlitebrowser -bundle-non-qt-libs -verbose=3
```

Comment:
Still seems to be failing, but the error message from the raw log seems to indicate it's due to the process being killed because of an oversized log file.

Guessing the "-verbose=3" should probably be removed?